packages/upset/src/components/Columns/Attribute/AttributePlots/StripPlot.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import { useRecoilValue } from 'recoil';
+import { ScaleLinear } from 'd3-scale';
+import { Aggregate, SixNumberSummary, Subset } from '@visdesignlab/upset2-core';
+import { dimensionsSelector } from '../../../../atoms/dimensionsAtom';
+import { visibleAttributesSelector } from '../../../../atoms/config/visibleAttributes';
+
+type Props = {
+  scale: ScaleLinear<number, number, never>;
+  values: number[];
+  attribute: string;
+  summary: SixNumberSummary;
+  isAggregate: boolean;
+  row: Subset | Aggregate;
+};
+
+// Dot plot component for the attributes plots
+export const StripPlot: FC<Props> = ({
+  scale, values, attribute, summary, isAggregate, row,
+}) => {
+  const dimensions = useRecoilValue(dimensionsSelector);
+  const attributes = useRecoilValue(visibleAttributesSelector);
+
+  if (summary.max === undefined || summary.min === undefined || summary.first === undefined || summary.third === undefined || summary.median === undefined) {
```

Comment:
More future-proof: 
```suggestion
  if (Object.keys(summary).some((key) => key === undefined)) {
```

Reply:
But also, where are these summary fields even used?

Reply:
There was an edge case for these breaking if undefined at some point, but it may have been covered elsewhere. I haven't been able to recreate that error so I have removed these from the props of both dot and strip plots. If we come across an error involving attribute load order this may be the place to look.